### DISCUSSION TITLE
How to use Kitty on Windows in a semi-cursed way

### DISCUSSION THREAD

**Discussion by jgregoire** [kovidgoyal/kitty#7054] [Show and tell]
![kitty_with_images](https://github.com/kovidgoyal/kitty/assets/915427/bd9fde64-fc02-44b3-9949-a03ff7b69367)

I wanted to use nvim+neorg with image rendering on Windows (which I have to use for my job). What you see here is Kitty running on WSL2, displaying an X window fully integrated with the rest of the Windows desktop environment.

I even have an icon pinned to the taskbar to launch kitty.

I followed these guides:

- https://learn.microsoft.com/en-us/windows/wsl/tutorials/gui-apps
- https://github.com/danielbisar/settings/blob/main/guides/kitty-on-windows-with-wsl2.md

Performance is great, and the window behaves well with FancyZones.

**Comment by anishkumar127** [comment]
@jgregoire  i need help im getting below error in windows 11 wsl2 how can i resolve this please let me know.

[080 07:47:20.913752] [glfw error 65544]: Wayland: Failed to connect to display
GLFW initialization failed

**Reply by rachartier** [reply]
You may need to run this command: `ln -sf /mnt/wslg/runtime-dir/wayland-* $XDG_RUNTIME_DIR/`
